src/public/video.ts

```diff
@@ -66,6 +78,25 @@ export namespace video {
     EffectDisabled,
   }
 
+  export interface PersonalizedEffect {
+    /**
+     * personalized effect id
```

Comment:
NIT: Capitalize

Reply:
Same below; please remember that these comments become part of our public docs once you commit your changes

Reply:
Updated